yap.sh

```diff
@@ -21,6 +21,7 @@ Options:
   -v, --interactive       Run everything interactively.
       --invidious=LINK    Specify invidious instace (default: https://yt.artemislena.eu)
       --track=NUM         Specify track number
+  --version               Show version
```

Comment:
Just a small adjustment: Could you align the string here to the other long options (see e.g. --play)? Thanks :)